source/code-generation/query-converter/enable-query-converter.txt

```diff
@@ -36,11 +36,27 @@ Steps
 
    .. step:: Read the user agreement
 
+   .. step:: (Optional) Disable telemetry
```

Comment:
Can we make the heading "Review telemetry preferences" or similar? We don't want to encourage disabling, so I'd like the language to be softer.